src/Compilers/CSharp/Test/Emit3/Semantics/ExtensionTests.cs

```diff
@@ -19279,6 +19329,9 @@ static class E
             // (1,9): error CS8130: Cannot infer the type of implicitly-typed deconstruction variable 'y'.
             // var (x, y) = new C();
             Diagnostic(ErrorCode.ERR_TypeInferenceFailedForImplicitlyTypedDeconstructionVariable, "y").WithArguments("y").WithLocation(1, 9),
+            // (1,14): error CS1061: 'C' does not contain a definition for 'Deconstruct' and no accessible extension method 'Deconstruct' accepting a first argument of type 'C' could be found (are you missing a using directive or an assembly reference?)
+            // var (x, y) = new C();
+            Diagnostic(ErrorCode.ERR_NoSuchMemberOrExtension, "new C()").WithArguments("C", "Deconstruct").WithLocation(1, 14),
```

Comment:
This looks like a duplicate diagnostic for the same thing as the one below.

Reply:
I agree, it does seem redundant. The PROTOTYPE comment above should cover that (we'll need to revisit pattern-based scenarios).